### ISSUE TITLE
Unknown object property "link_redirects" 

### ISSUE THREAD

**Issue by elias506** [VKCOM/vk-api-schema#211]
Hello!

I stumbled upon the property `link_redirects`  in `account_info` object  in objects.json:

```
"link_redirects": {
          "type": "object"
        },
```

Actual question, what does that means?

**Comment by SevereCloud** [comment]
https://github.com/VKCOM/api-schema-typescript-generator/issues/19

Можно привести к такому

```go
type AccountInfo struct {
  LinkRedirects map[string]interface{} `json:"link_redirects"` // Ну или map[string]any если версия go >= 1.18
}
```

На деле api возвращает объект со строками, поэтому у себя в [vksdk](https://github.com/SevereCloud/vksdk) я использую `map[string]string`

https://github.com/SevereCloud/vksdk/blob/e983680a2c27787d571362892aba53ae5e86d59d/object/account.go#L110-L112

**Comment by elias506** [comment]
Да, я понимаю, что можно просто использовать `map[string]string`, но мне нужно конкретно тип поля
